nasa-gibs/worldview
# Update MODIS_Combined_L3_IGBP_Land_Cover_Type_Annual.md to c6.1

## Description

Fixes #WV-3316.

Update layer description to describe c6.1 for MODIS Land Cover Type. 

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-77.57559342918314,-26.713983125513707,73.69370567963763,47.70148103309424&i=1&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,MODIS_Combined_L3_IGBP_Land_Cover_Type_Annual,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2020-09-30-T15%3A11%3A52Z)
2. Verify expected result - check that layer description describes collection 6.1 and link points to collection 6.1; verify that the imagery changes annually from 2001 to 2023.


@nasa-gibs/worldview
